src/bootstrap.js

```diff
@@ -14,10 +18,25 @@ function install() {} // eslint-disable-line no-unused-vars
 function uninstall() {} // eslint-disable-line no-unused-vars
 
 function startup() { // eslint-disable-line no-unused-vars
+  Services.prefs.setBoolPref(UA_ENABLE_PREF_NAME, true);
```

Comment:
If code forces on in m-c, then all users will be affected even in release channel. Could we consider force on in nightly but off in other versions? @miketaylr ? any comment?